parts/main.fodt

```diff
@@ -264148,7 +264148,7 @@ Updated with AFR/TSA Rev-D comments and new  keywords.</dc:description><meta:ini
        <text:p text:style-name="Table_20_Contents"><text:s text:c="2"></text:s>Document Name</text:p>
       </table:table-cell>
       <table:table-cell table:style-name="Table1883.B22" table:number-columns-spanned="4" office:value-type="string">
-       <text:p text:style-name="P10483"><text:file-name text:display="full">davs://blattms@storage.opm-op.com/remote.php/dav/files/blattms/opm-op/Projects/P20-07 Equinor Maintenance/Reference Manual/2023-04 Manual/OPM_Flow_Reference_Manual_2023-04_Rev-0.odt</text:file-name></text:p>
+       <text:p text:style-name="P10483">OPM_Flow_Reference_Manual_2023-04_Rev-A.fdot</text:p>
```

Comment:
@blattms Thanks for the update. Could we add a space in front of "github", see screenshot below. Also, the document name: should it be `main.fodt`? Or `OPM_Flow_Reference_Manual_2024-02_Rev-A.fodt` (change year to 2024 and fdot to fodt). Also, when I click the "github" link nothing happens.
![image](https://github.com/OPM/opm-reference-manual/assets/6708379/13a3e8bb-3868-4232-a6ae-e0f5cbc8d3e5)

Reply:
github is fixed.
I think document name "main.fodt" is not very speaking. I kept the old naming convention here OPM_Reference_Manual_<release-version>.fodt. I will correct the extensio then to have OPM_Reference_Manual_2023-10_Rev-A.fodt

Reply:
Done.

Reply:
Thanks for the update! When I CTRL-click the "github" link, nothing happens. Should it open in the default browser on my computer, or is this only the case after we have exported to PDF?

![image](https://github.com/OPM/opm-reference-manual/assets/6708379/780dba2a-d9a4-4c94-8988-f98fffa6abf9)

Reply:
> or is this only the case after we have exported to PDF?

Checked after exporting to PDF, and it works when I click the link in the PDF file.